case_study1_airline_safety.Rmd

```diff
@@ -0,0 +1,106 @@
+
+    ```{r include = FALSE}
+    ## need to find the best way to let students download the data 
+    data=read.csv2("~/Documents/TA_Econometrics/Case_studies/airline_safety/data-master/airline-safety/airline-safety_long.csv", sep = ";")
+    ```
+    
+## Exploring the data
+
+The dataset contains the safety records of major commercial airlines over the past 30 years (1985 to 2014). The period has been break down into two halves: from 1985 to 1999, and from 2000 to 2014.
+
+1. First look at the dataset
+    1. What are the variables names and types (categorical, numerical, ...) included in the data ?
+    ```{r}
+    #names(data)
+    str(data)
+    ```
+    1. What is the number of observations in total?
+    ```{r}
+    dim(data)[1]
+    ```
+    1. What defines an observation in our case (no need to code) ? 
+    
+1. Having a  closer look
+    1. How does the dataset look like ? Have a look to the first 5 observations, and the bottom 5. 
+    ```{r}
+    head(data)
+    tail(data)
+    ```
+    1. What are the different values of the "type" variable ? Same question for the "period" variable.
+    ```{r}
+    table(data$type, useNA = "ifany")
+    table(data$period, useNA = "ifany")
+    ```
+    1. Is there any NA's in the dataset ?
+    ```{r}
+    sum(is.na(data))
+    apply(data, 2, function(x) any(is.na(x)))
+    ```
+    1. The variable "avail_seat_km_per_week" corresponds to the number of seats multiplied by the number of kilometers the airline flies in a week. In your opinion, why this variable could be important for future analysis ?
+
+## Analysis
+
+1. What are the mean value of the number of incidents by type and period ? Same for the standard deviation ?
+    ```{r}
+    
+    data %>%
+      group_by(period, type) %>% 
+      summarise(mean_fatal_by_period = mean(value), 
+                sd_fatal_by_period = sd(value))
+    ```
+1. Interpretation : Overall, what happened between the two periods? What can we say about the relative value of the standard deviation compared to the mean? Do you find the mean value meaningfull in this case? 
+
+1. Propose a vizualization showing the evolution of the number of fatal accidents between the two periods. 
+    ```{r}
+    ggplot(subset(data, type != "fatalitie"), aes(x = period, y = value))+
+      geom_boxplot()+
+      facet_wrap(~type)
+      
+    
+    ```
+1. Over 2000-2014 period, what is the top 3 companies (meaning the worst companies) in terms of fatalities?
+    ```{r}
+    data %>% 
+      filter(type == "fatalitie" & period == "2000_2014") %>%
+      arrange(desc(value)) %>%
+      head(3)
+    ```
+1. Do the same taking into account the "avail_seat_km_per_week" variable? 
+    ```{r}
+    data %>% 
+      filter(type == "fatalitie" & period == "2000_2014") %>%
```

Comment:
really `fatalitie` ? is that a french dataset? you write `fatality` or `fatalities` in english.

Reply:
The guy who reshaped the dataset (me) is indeed french :), actually it was a typo in the code I remove one character ("s") by error